examples/main.rs

```diff
@@ -69,12 +69,14 @@ fn main() {
                     s += "\n";
                     for x in 0..grid.size()[1] {
                         if let Some(x) = grid.get_item([x, y]).determined_value {
-                            s += x.to_char().to_string().as_str();
+                            s += &String::from(x);
```

Comment:
x.into() is likely better.

Reply:
With the default being `From<Tile>` I think this okay now

Reply:
True